modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submit.multiplepasses.unit.cy.js

```diff
@@ -0,0 +1,89 @@
+import { lib } from './lib.js';
+import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
+
+describe('Submission with multiple passes', () => {
+  const form = {
+    date: '1950-01-02',
+    crops: ['BEAN', 'CARROT'],
+    location: 'ALF',
+    beds: ['ALF-1', 'ALF-3'],
+    areaSeeded: 50,
+    seedApplicationEquipment: ['Tractor'],
+    seedIncorporationEquipment: ['Rake'],
+    seedApplicationDepth: 6,
+    seedApplicationSpeed: 5,
+    seedApplicationPasses: 3,
+    seedIncorporationDepth: 8,
+    seedIncorporationSpeed: 3,
+    seedIncorporationPasses: 3,
+    winterKill: true,
+    winterKillDate: '1950-12-31',
+    comment: 'Multi-pass test',
+  };
+  let results = null;
+
+  before(() => {
+    const timeout = { timeout: 30000 };
+    cy.wrap(farmosUtil.getBedNameToAssetMap(), timeout)
+      .then(() => {})
```

Comment:
Most of the statements below load maps of information that were used in the original `submit` test.  But it doesn't look like those maps are being used in this test.  They can be removed.